### PR TITLE
Subscription to notification

### PR DESCRIPTION
**Pull request by VladSemenyuk**

### CODE REVIEW

**Comment by pvysh9** [comment]
Reviewed

File: src/components/can_cooperation/src/command/get_interior_vehicle_data_request.cc [hunk 1]
```diff
@@ -63,6 +64,27 @@ void GetInteriorVehicleDataRequest::Execute() {
...
+
+    if (!app) {
+      LOG4CXX_ERROR(logger_, "Application doesn't registered!");
+      SendResponse(false, result_codes::kApplicationNotRegistered, "");
+      return;
+    }
+
+    CANAppExtensionPtr extension = application_manager::AppExtensionPtr::
+        static_pointer_cast<CANAppExtension>(app->QueryInterface(
+            CANModule::instance()->GetModuleID()));
```
Comment (bonewell) [inline src/components/can_cooperation/src/command/get_interior_vehicle_data_request.cc]: I think you could use BaseCommandRequest::GetAppExtension

**Comment by bonewell** [comment]
Reviewed.

File: src/components/can_cooperation/src/command/get_interior_vehicle_data_request.cc (hunk 1 above)
Comment (VladSemenyuk) [inline src/components/can_cooperation/src/command/get_interior_vehicle_data_request.cc]: Ok

**Pull request merged**